### PR TITLE
Create professional readme file

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
Here, I added a professional readme file with live demo from the develop branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,271 @@
...
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Most Visitied Countries] <a name="about-project"></a>
+
+**[Most Visitied Countries]** is a project that uses the API for its data resource.
```
Comment (sergio-usma) [inline README.md]: I think you can add more information to the description. For example:

_This project is a web application that provides information about different countries. It allows users to explore details such as the continent, capital, area, and population of various countries. Users can also view comments and add their own insights about specific countries. The application utilizes the Involvement API to enable features like liking countries and commenting. It offers a user-friendly interface with pop-up windows to display country information and interact with the community through comments. The project aims to provide an engaging platform for users to learn and share knowledge about different countries._

File: README.md [hunk 2]
```diff
@@ -1 +1,271 @@
...
+
+- 👤 GitHub: [@Anwar Hussaini](https://github.com/M-Anwar-Hussaini)
+- 👤 Twitter: [@MAnwarHussaini](https://twitter.com/MAnwarHussaini)
+- 👤 LinkedIn: [Mohammad Anwar Hussaini](https://www.linkedin.com/in/mohammad-anwar-hussaini-876638267/)
+
+👤 **Sergio Usma**
+
+- GitHub: [@sergio-usma](https://github.com/sergio-usma)
+- Twitter: [@VonUsma](https://twitter.com/vonusma)
+- LinkedIn: [Sergio Andres Usma](https://www.linkedin.com/in/sergiousma/)
```
Comment (sergio-usma) [inline README.md:223]: The links and information are correct!

File: README.md [hunk 3]
```diff
@@ -1 +1,271 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+**I would like to thank the following individuals and organizations for their contribution to this project.**
+
+- 🙏 [**Microvere**](https://www.microverse.org/?grsf=mohammad-a-nbtazu) for its arrangement and supports to all Micronauts and me.
```
Comment (sergio-usma) [inline README.md]: I think you could add here something like:

_I would like to express my heartfelt gratitude to Microverse for the invaluable learning experience they have provided. The supportive community, dedicated mentors, and remote collaboration opportunities have enhanced my technical skills and prepared me for real-world projects. I extend my appreciation to the mentors and staff members for their guidance and support. The friendships and knowledge sharing within the Microverse community have made this journey truly rewarding._

**Review by sergio-usma — APPROVED** [review]
# STATUS: APPROVED ✅ 🍾
Hi @M-Anwar-Hussaini 
Your project is complete! There is nothing else to say other than... it's time to merge it!
Congratulations! 🎇

# To highlight 🍾
All linters are passing ✅
Good commit history ✅
Good PR description ✅ 

# Optional suggestions

- [ ] Please see the comments to add more information in the description and acknowledgments sections.

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (M-Anwar-Hussaini) [reply README.md]: Hi @sergio-usma. Thanks a lot for your feedback. I'll add the descriptions. It is really great.

**Pull request merged**